src/main/java/com/conveyal/r5/profile/Propagater.java

```diff
@@ -25,17 +25,21 @@
     /** Times at targets using the street network */
     public final int[] nonTransferTravelTimesToTargets;
 
+    /** The maximum allowable travel time in this regional analysis, to avoid propagating things that won't be useful */
+    public final int cutoffSeconds;
```

Comment:
The expression "maximum allowable travel time" is not clear. At first I took this to mean the maximum representable or displayable travel time, but reading the code I think this refers to the accessibility indicator travel time threshold value. 

The phrase "propagating things that won't be useful" could be clarified as well. "Things" means "times", and I wouldn't say those times "won't be useful", it's more that the action of propagation is unnecessary, because propagation can only increase travel time, and our end result is just whether travel times are over or under the threshold, not the travel times themselves. This allows an optimization.